python/test/test_polars.py

```diff
@@ -71,3 +72,11 @@ def test_lazyframe_ffi():
     lf_domain, lf = example_lf()
     t_ident = (lf_domain, dp.symmetric_distance()) >> dp.t.then_identity()
     assert t_ident(lf).collect().equals(lf.collect())
+
+def test_expr_ffi():
+    """ensure that lazyframes can be passed to/from Rust"""
+    pl = pytest.importorskip("polars")
+    lf_domain, lf = example_lf()
+    expr_domain = dp.expr_domain(lf_domain, "select")
+    t_ident = (expr_domain, dp.symmetric_distance()) >> dp.t.then_identity()
```

Comment:
It's not obvious that the assertion depends on these lines. Maybe stronger assertions? Or more explanation? Or drop these?

Reply:
The primary purpose of this is to ensure that the FFI is working properly. The identity function tests both data loading and unloading.